assets/js/scroll-to.js

```diff
@@ -0,0 +1,38 @@
+export default () => {
+  window.addEventListener("load", scrollTo, { passive: true });
+};
+
+const scrollTo = () => {
+  window.requestAnimationFrame(() => {
+    const initialElToScrollTo = document.querySelector("[data-scroll-to]");
```

Comment:
I searched so freakin hard for where the heck this element was and never found it, hence I'd mistakenly deleted this file! I _still_ couldn't find it for the past ten minutes... finally though, I realize it's here:

https://github.com/mbta/dotcom/blob/eba1e9ee40297c4ae614736dbf728cf2e1223be6/lib/dotcom_web/templates/schedule/_timetable.html.eex#L35